src/app/theme.cpp

```diff
@@ -48,6 +48,8 @@ Theme::Theme() : QObject(qApp), palette()
     QFontDatabase::addApplicationFont(":/fonts/Montserrat/Montserrat-LightItalic.ttf");
     QFontDatabase::addApplicationFont(":/fonts/Montserrat/Montserrat-Regular.ttf");
 
+    qApp->setFont(Theme::font_14);
```

Comment:
was this added in from a merge or something?

Reply:
Very likely